.gitignore

```diff
@@ -157,4 +157,6 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
\ No newline at end of file
+#.idea/
+
+prometheus_dev
```

Comment:
Its a bit non standard directory. Unless you have development instructions that use this, its better to add a gitignore in the directory and ignore everything inside it.